graylog2-web-interface/src/logic/telemetry/TelemetryProvider.tsx

```diff
@@ -60,6 +60,8 @@ const TelemetryProvider = ({ children }: { children: React.ReactElement }) => {
   const posthog = usePostHog();
   const theme = useTheme();
 
+  const isPosthogLoaded = useCallback(() => !!(posthog && posthog.__loaded), [posthog]);
```

Comment:
What about doing something like 
```suggestion
  const isPosthogLoaded = useCallback(() => (posthog?.__loaded === true), [posthog]);
```
Maybe that is more explicit?

Reply:
Committing your suggestion, that's more explicit.